examples/webgl_materials_instance_uniform_nodes.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - material instance uniform</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl material instance uniform
+		</div>
+
+		<script type="importmap">
+		{
+			"imports": {
+				"three": "../build/three.module.js"
+			}
+		}
+		</script>
+		<script type="module">
+
+			import * as THREE from 'three';
+
+			import Stats from './jsm/libs/stats.module.js';
+
+			import { nodeFrame } from './jsm/renderers/webgl/nodes/WebGLNodes.js';
+			import { NodeUpdateType } from './jsm/renderers/nodes/core/constants.js';
+			import Node from './jsm/renderers/nodes/core/Node.js';
+			import ColorNode from './jsm/renderers/nodes/inputs/ColorNode.js';
+
+			class InstanceUniformNode extends Node {
+
+				constructor() {
+
+					super( 'vec3' );
+
+					this.updateType = NodeUpdateType.Object;
+
+					this.inputNode = new ColorNode();
+
+				}
+
+				update( frame ) {
+
+					const rendererState = frame.renderer.state;
+					const mesh = frame.object;
+
+					const meshColor = mesh.color;
+
+					this.inputNode.value.copy( meshColor );
+
+					// force refresh material uniforms
+					rendererState.useProgram( null );
```

Comment:
Um, it's not ideal that `WebGLState.useProgram()` is used here. The method is considered to be private and should not be used outside of the renderer.

I guess we need a different approach in the future to trigger a refresh of the material's uniforms.